src/passes/SimplifyGlobals.cpp

```diff
@@ -659,10 +660,11 @@ struct SimplifyGlobals : public Pass {
         // This is the init of a passive segment, which is null.
         return;
       }
-      if (auto* get = init->dynCast<GlobalGet>()) {
+      for (auto** getp : FindAllPointers<GlobalGet>(init).list) {
```

Comment:
As a separate cleanup it might be nice to use the template parameter on `FindAllPointers` in the type of its `list` to avoid the need for a cast below.

Reply:
I'll try, though I have a vague memory of that not working for a sad C++ey reason...